src/openrpc/player.json

```diff
@@ -0,0 +1,814 @@
+              "value": "123"
+            },
+            {
+              "name": "status",
+              "value": {
+                "mediaSessionId": "sess2",
+                "state": "BLOCKED",
+                "blockedReason": "CONTENT_NOT_FOUND"
+              }
+            }
+          ],
+          "result": {
+            "name": "providedResponse",
+            "value": null
+          }
+        }
+      ]
+    },
+    {
+      "name": "provideProgress",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-provides": "xrn:firebolt:capability:player:base"
+        }
+      ],
+      "summary": "Tells listeners the current progress of a player",
+      "params": [
+        {
+          "name": "playerId",
+          "summary": "The id of the player the progress is for",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "progress",
+          "summary": "The progress",
+          "schema": {
+            "$ref": "#/components/schemas/PlayerProgress"
+          },
+          "required": true
+        }
+      ],
+      "result": {
+        "name": "providedResponse",
+        "schema": {
+          "const": null
+        }
+      },
+      "examples": [
+        {
+          "name": "Default example #1",
+          "params": [
+            {
+              "name": "playerId",
+              "value": "123"
+            },
+            {
+              "name": "progress",
+              "value": {
+                "speed": 1,
+                "startPosition": 0,
+                "position": 5000,
+                "endPosition": 3600000,
+                "liveSyncTime": "2021-04-23T18:25:43.511Z"
+              }
+            }
+          ],
+          "result": {
+            "name": "providedResponse",
+            "value": null
+          }
+        }
+      ]
+    }
+  ],
+  "components": {
+    "schemas": {
+      "PlayerLoadRequest": {
+        "title": "PlayerLoadRequest",
+        "type": "object",
+        "required": [
+          "playerId",
+          "locator"
+        ],
+        "properties": {
+          "playerId": {
+            "description": "The id of the player instance to load the locator into",
+            "type": "string"
+          },
+          "locator": {
+            "description": "A locator to the content to load",
+            "type": "string",
+            "format": "uri"
+          },
+          "metadata": {
+            "description": "An opaque set of metadata that will be relevant to the player.",
+            "type": "object"
+          },
+          "autoPlay": {
+            "description": "Start playing immediately on load without needing to call Player.play",
+            "type": "boolean"
+          }
+        },
+        "examples": [
+          {
+            "playerId": "app1:123",
+            "locator": "https://cdn.rdkcentral.com/media/assets/asset.hls"
```

Comment:
Can this also be used for HDMI playback?

Reply:
possibly, but needs some more thought if HDMI is broken out into its own module